04_HowTos/FileService/How_to_use_the_File_Services.ipynb

```diff
@@ -17,20 +18,61 @@
    "source": [
     "# How to Use the Data Lab Public File Services\n",
     "\n",
-    "_Mike Fitzpatrick and Glenn Eychaner_\n",
+    "*Revised:  Sept 01, 2020*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Table of contents\n",
+    "* [Summary](#summary)\n",
+    "* [Disclaimer & Attribution](#attribution)\n",
+    "* [Imports & setup](#import)\n",
+    "* [Listing another user's public file space](#listspace)\n",
+    "* [Listing all available public file spaces](#listall)\n",
+    "* [Example: using the **SDSS DR14** public file service](#exsdss)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"summary\"></a>\n",
+    "# Summary\n",
+    "\n",
+    "Files in the virtual storage VOSpace are usually identified via the prefix \"*vos://*\". This shorthand identifier is resolved to a user's home directory of the storage space in the service.  \n",
     "\n",
-    "*Revised:  Jan 03, 2019*\n",
+    "If the \"*vos://*\" prefix is instead the name of another user (e.g. \"*geychaner://*\"), and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a \"*/public*\" directory to facilitate file sharing (e.g. '*geychaner://public/foo.fits*' will access the '*foo.fits*' file from user '*geychaner*'). Users can make any file (or directory) public by moving it to (or creating a link in) their \"/public\" directory.\n",
```

Comment:
I propose to use a different demo account here. Glenn Eychaner isn't working at DL anymore. May I suggest "demo00" as the standard strawperson user? I'll send you the password for it in Slack.

Reply:
Done.